tests/conftest.py

```diff
@@ -0,0 +1,240 @@
+"""Common fixtures."""
+import copy
+from datetime import date
+from typing import Any, Dict, Iterator
+
+import pytest
+import respx
+
+from rtbhouse_sdk.client import build_base_url
+
+
+@pytest.fixture()
```

Comment:
nit:
These parentheses are redundant
```suggestion
@pytest.fixture
```